.travis.yml

```
@@ -24,7 +24,7 @@ cache:
 install: true
 
 script:
-  - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent verify sonar:sonar -P deb
+  - mvn "${MVN_ARGS}" clean org.jacoco:jacoco-maven-plugin:prepare-agent verify sonar:sonar -P deb
```

No need for quotes:

```suggestion
  - mvn ${MVN_ARGS} clean org.jacoco:jacoco-maven-plugin:prepare-agent verify sonar:sonar -P deb
```

See:

* https://travis-ci.com/retest/rebazer/builds/130372562#L507
* https://github.com/retest/recheck-web/blob/master/ci/script.sh#L14